src/modlistsortproxy.cpp

```diff
@@ -177,21 +177,14 @@ bool ModListSortProxy::lessThan(const QModelIndex &left,
       }
     } break;
     case ModList::COL_CONTENT: {
-      std::vector<ModInfo::EContent> lContent = leftMod->getContents();
-      std::vector<ModInfo::EContent> rContent = rightMod->getContents();
-      if (lContent.size() != rContent.size()) {
-        lt = lContent.size() < rContent.size();
+      unsigned int lValue = 0;
+      unsigned int rValue = 0;
+      for (int content : leftMod->getContents()) {
+        lValue += 2U << static_cast<unsigned int>(content);

```

Comment:
I don't think it's the same still but I also don't think the current sorting is much good

Reply:
What do you mean? This should be equivalent to version on `master`.

Reply:
I think the original was missing an else after  `lt = lContent.size() < rContent.size();`

Reply:
It would make a weird sort I think, and maybe a non-valid one so I'm not sure...